docs/getting_started_guide/SIMP_on_Amazon_Web_Services.rst

```diff
@@ -0,0 +1,73 @@
+.. _gsg-simp_on_aws:
+
+SIMP on Amazon Web Services
+===========================
+
+This chapter provides notes and guidance on using the official SIMP Amazon
+Machine Image (AMI) to run the SIMP server in the cloud. 
+
+The SIMP AMI is built from the SIMP ISO, so much of the information contained 
+in the ISO installation section :ref:`ig-simp-server-installation` applies 
+here. 
+
+Provision a New EC2-Instance
+----------------------------
+
+To provision a new ec2-instance in the AWS cloud running the official SIMP AMI,
+follow these steps:
+   - Launch a new instance in the normal way, and navigate to the AWS 
+     Marketplace tab when prompted to choose a Machine Image. 
+   - Search the marketplace for the SIMP AMI, and locate the official published
+     SIMP AMI. You can also find the AMI by the following ID: ``ami-efbf8ef9``
+   - Your ec2-instance should be **at least** ``t2.medium``, with 2 cpus and 
+     4GB of memory. Less than 4GB of memory will significantly slow down the 
+     bootstrapping process, and might cause problems in the future. See the 
+     AWS documentation_ for details on instance sizes, and the Puppet 
+     `Installation Guide`_ for details on hardware requirements. 
+   - When selecting security group rules for your instance, ensure that you 
+     have the necessary ports open. At the very least, you need to ensure that
+     you can SSH into the instance after it is running (port 22). 
```

Comment:
We should explicitly mention that they need to configure the security group to allow port 8140 from /their/ internal subnets only, as this isn't the same as a normal airgapped install